### ISSUE TITLE
Should output error `Callable` misuse

### ISSUE THREAD

**Issue by tamuhey** [microsoft/pyright#1365]
**Describe the bug**

`Callable[[str]]` is misuse and exec fails, but pyright doesn't output any error.

**To Reproduce**

```python
from typing import Callable

a: Callable[[str]] # SHOULD output error here
```

**Expected behavior**

Should output Error

pyright version: 1.1.101

**Comment by erictraut** [comment]
Thanks for the suggestion. This will be addressed in the next release of pyright and pylance.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.102, which I just published. It was already included in the latest version of pylance.
